docs/content/config/advanced/podman.md

```diff
@@ -96,6 +96,94 @@ docker compose up -d mailserver
 docker compose ps
 ```
 
+### Dockermailserver as a rootless Quadlet
+
+Podman generate systemd is now deprecated and has been integrated into Quadlet. A quadlet is a ini-file with the same structure as a systemd unit file. Writing a Quadlet and placing it on your system will generate a systemd-service on your machine for containers, pods, volumes, networks, kubes, images and builds. Quadlets created as root will then auto-start.
+
+To make a rootless Quadlet auto-start make sure your user is *lingering*.
+
+```bash
+loginctl enable-linger user
+```
+
+Forcing a daemon-reload when a Quadlet-file is present in any of the accepted directories for Quadlets, will automatically generate a systemd-service that will auto-start at boot.
+
+```bash
+systemctl --user daemon-reload
+```
+
+Quadlet files for non-root users can be placed in the following directories
+
+- $XDG_RUNTIME_DIR/containers/systemd/
+- $XDG_CONFIG_HOME/containers/systemd/ or ~/.config/containers/systemd/
+- /etc/containers/systemd/users/$(UID)
+- /etc/containers/systemd/users/
+
+#### Example Quadlet file
+
+We have to use the .container extension for the quadlet generator to pick up the service.
+Because dockermailserver uses multiple users inside the container, we will either have to use our own user as root, resulting in our e-mails being owned by a subuid. Alternatively, using UIDMap we can map our rootless user to UID 5000 in the container who owns our e-mails. Using UIDMap also maps root user 0 inside the container to an available sub-uid of our rootless user. Otherwise the container will not have permission to configure itself.
```

Comment:
```suggestion
Because docker-mailserver uses multiple users inside the container, we will either have to use our own user as root, resulting in our e-mails being owned by a subuid. Alternatively, using UIDMap we can map our rootless user to UID 5000 in the container who owns our e-mails. Using UIDMap also maps root user 0 inside the container to an available sub-uid of our rootless user. Otherwise the container will not have permission to configure itself.
```